docs/overview/coords.md

```diff
@@ -148,7 +148,7 @@ For example to align a label above a button and center the label is horizontally
 lv_obj_align_to(label, btn, LV_ALIGN_OUT_TOP_MID, 0, -10);
 ```
 
-Not that - unlike with `lv_obj_align()` - `lv_obj_align_to()` can not realign the object if its coordinates or the reference object's coordinates changes. 
+Note that - unlike with `lv_obj_align()` - `lv_obj_align_to()` can not realign the object if its coordinates or the reference object's coordinates changes. 
```

Comment:
In previous versions this was possible, right?

Reply:
In v7 there was `obj_realign` that could realign on obj's size change. But didn't work on parent's size change.

Reply:
> In v7 there was `obj_realign` that could realign on obj's size change. But didn't work on parent's size change.

thanks, oops, so then what is the `obj_realign` replacement?

Reply:
Objects aligned on their parent with `lv_obj_align` are realigned automatically but there is no replacement when the object is aligned to another object using `lv_obj_align_to`. In such a case probably it's better to use a [layout](https://docs.lvgl.io/8.0/layouts/index.html).